src/chart/bar/BarView.ts

```diff
@@ -996,6 +996,8 @@ function updateStyle(
         );
     }
 
+    cursorStyle && label.attr('cursor', cursorStyle);
```

Comment:
Many other series may have similar problems. Please consider setting cursor information to labels in LabelManager. You may refer to [updateBlend](https://github.com/apache/echarts/blob/b73005501cdee11e0223722b160289e5a4a88676/src/core/echarts.ts#L2200) for reference.

Reply:
Now setting cursor in `processLabelsOverall` of LabelManager 😂